packages/hosted-form-v2/src/hosted-form-service.ts

```diff
@@ -23,14 +24,17 @@ export default class HostedFormService {
         }
     }
 
-    async submitManualOrderPayment(data: HostedFormManualOrderData): Promise<void> {
+    async submitManualOrderPayment(
+        data: HostedFormManualOrderData,
+    ): Promise<HostedInputSubmitManualOrderSuccessEvent | void> {
         const form = this._hostedForm;
 
         if (!form) {
             throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
         }
 
         await form.validate();
-        await form.submitManualOrderPayment({ data });
+
+        return form.submitManualOrderPayment({ data });
```

Comment:
This method is still `async`, also what are the implications for removing the await call here?

Reply:
The promise will be passed to the manual-order-ui side and resolved there.
I'll update the PR to keep the previous logic, which is resloving the promise here.